extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -122,7 +122,11 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 				text: constants.DATABASE_BACKUP_CHECKSUM_INFO_TEXT,
 				style: 'information',
 				width: WIZARD_INPUT_COMPONENT_WIDTH,
-				CSSStyles: { ...styles.BODY_CSS }
+				CSSStyles: { ...styles.BODY_CSS },
+				links: [{
+					text: constants.READ_MORE,
```

Comment:
I recommend adding an ariaLabel property to the link that describes where the link will go... this will give the screen reader more context than "read more"...  something like the title of the article can be helpful.

Reply:
maybe add more context to arial label to read as:
'Learn more about, enabling or disabling checksums ...'

Reply:
👍added.

Reply:
@junierch since the need for checksum has been removed, do you want to change this PR/task and remove the infobox with checksum information instead.

Reply:
as discussed offline, I made this change in my own PR before I saw Brian's latest comment: https://github.com/microsoft/azuredatastudio/pull/20723/commits/fb6313eb137129b61506cb088b5340b1a610d179

We could abandon this PR and take this change there, up to you